### DISCUSSION TITLE
ReverseMap with type converter

### DISCUSSION THREAD

**Discussion by saiprabhur** [LuckyPennySoftware/AutoMapper#3897] [Ideas]
### Source/destination types

```csharp
// Put your source/destination types here
```

### Mapping configuration
  var config = new MapperConfiguration(cfg =>
            {
               
                CreateMap<AddSystemRequest, SystemDTO>().ConvertUsing(new SystemConverter());
                CreateMap<AddModuleRequest, ModuleDTO>().ConvertUsing(new SystemConverter());
            });

### Version: x.y.z
 
10.1.1
### Expected behavior

expecting reversemap possible for itypeconverter ?
### Steps to reproduce

```csharp
// Your calls to Mapper.Map or ProjectTo here, with source/destination objects constructed
```

**Comment by lbargaoanu** [comment]
You can write your own extension method for that. Other than `ReverseMap`, nothing makes sense after `ConvertUsing`.
